### DISCUSSION TITLE
part-ii/2017-88/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#407] [Question comments]
Part II 2017 Logic and Set Theory: Paper 4, Section II, H 

https://questions.tripos.org/part-ii/2017-88/

**Comment by artie2000** [comment]
All but last part are bookwork.

<details>
<summary>Last part:</summary>

Yes: $C_{\omega_1}=C_{\omega_1^+}$

Indeed, if $x\in C_{\omega_1}$ is countable, then, for $y\in x$, there is a countable ordinal $\alpha_y$ such that $y\in C_{\alpha_y}$. Let $\gamma=\sup_{y\in x}\alpha_y$; since $x$ and each $\alpha_y$ are countable, so is $\gamma$. Since $C_{\alpha}\subsetneq C_{\beta}$ for $\alpha<\beta$, $x\subset C_\gamma$, and so $x\in C_{\gamma^+}$.

Note that $C_{\omega_1}$ is the set of hereditarily countable (HC) sets: that is, countable sets all of whose elements are themselves HC.

</details>
